src/angular/dialog/dialog-content-directives.ts

```diff
@@ -21,6 +21,11 @@ import { SbbDialogRef, _closeDialogVia } from './dialog-ref';
 /** Counter used to generate unique IDs for dialog elements. */
 let dialogElementUid = 0;
 
+const closeAriaLabel = () =>
```

Comment:
I'd prefer a constant instead of a "function". Or was there a reason against it?